web-registry/src/components/organisms/MediaForm.tsx

```diff
@@ -357,6 +358,8 @@ const MediaForm: React.FC<MediaFormProps> = ({ submit, initialValues }) => {
               </OnBoardingCard>
               <ProjectPageFooter
                 onSave={submitForm}
+                // onNext={() => null} // TODO route to /metadata
+                onPrev={() => navigate(`/project-pages/${projectId}/story`)}
```

Comment:
this can go either to /story or /description depending on whether we have some on chain credit class id or not